internal/pkg/manifest/validate.go

```diff
@@ -1411,6 +1412,9 @@ func (p PublishConfig) validate() error {
 
 // validate returns nil if Topic is configured correctly.
 func (t Topic) validate() error {
+	if t.Type != nil && !contains(aws.StringValue(t.Type), validTopicsTypeValues) {
+		return fmt.Errorf(`"type" value %q is not allowed; must be one of %s`, aws.StringValue(t.Type), english.WordSeries(validTopicsTypeValues, "or"))
```

Comment:
This sounds very like a general type of error. Can we make it a type? Like it happens when the value of a field is not in a valid list.

Reply:
Actually when the error occurs it looks something like `validate "topics[0]": "type" value "lifo" is not allowed; must be one of standard or fifo"` 
Does that look good to you?